client/my-sites/plans/plan-overview/plan-remove/index.jsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import Dispatcher from 'dispatcher';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Dialog from 'components/dialog';
+import { isInGracePeriod } from 'lib/plans';
+import notices from 'notices';
+import paths from '../../paths';
+import { fetchSitePlansCompleted } from 'state/sites/plans/actions';
+import wpcom from 'lib/wp';
+
+const PlanRemove = React.createClass( {
+	propTypes: {
+		plan: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired,
+		store: React.PropTypes.object.isRequired
+	},
+
+	getInitialState() {
+		return {
+			isCanceling: false,
+			showDialog: false
+		};
+	},
+
+	removePlan( closeDialog ) {
+		this.setState( { isCanceling: true } );
+
+		wpcom.undocumented().cancelPlanTrial( this.props.plan.id, ( error, data ) => {
```

Comment:
Sounds like a good idea and a great opportunity to get a better understanding of Redux. I'd like to tackle that in another pull request though so we can deploy this one as soon as possible. I'll create an issue to track that.